WORKSPACE

```diff
@@ -266,64 +266,11 @@ new_go_repository(
 ## Proxy build rules
 ##
 
-git_repository(
+http_file(
     name = "istio_proxy",
-    commit = "53fd02610571f8c05af2e50fea911f89f935e678",
-    remote = "https://github.com/istio/proxy",
-)
-
-load(
-    "@istio_proxy//:repositories.bzl",
-    "boringssl_repositories",
-    "protobuf_repositories",
-    "googletest_repositories",
+    url = "https://storage.googleapis.com/istio-build/proxy/envoy-alpha-eb8bfccc391e528fba316c4e081515eb18583e5c.tar.gz",
```

Comment:
worth adding a comment about the version of envoy (if there is a worthwhile distinction between alpha versions) that is retrieved at this location?